Networking/Networking/Mapper/StripeAccountMapper.swift

```diff
@@ -0,0 +1,47 @@
+import Foundation
+
+/// Mapper: Stripe Account
+///
+struct StripeAccountMapper: Mapper {
+
+    /// (Attempts) to convert a dictionary into an account.
+    ///
+    func map(response: Data) throws -> StripeAccount {
+        let decoder = JSONDecoder()
+
+        /// Needed for currentDeadline, which is given as a UNIX timestamp.
+        /// Unfortunately other properties use other formats for dates, but we
+        /// can cross that bridge when we need those decoded.
+        decoder.dateDecodingStrategy = .secondsSince1970
+
+        /// Detect the exceptional case where we got a response of data:[] indicating
+        /// that the plugin is active but the merchant has not on-boarded (and therefore
+        /// has no account.)
+        if let _ = try? decoder.decode(StripeNullAccountEnvelope.self, from: response) {
```

Comment:
nit: is this case an edge case, or could be quite common (like > 50%)? If it's an edge case, I wonder if we could try decoding with `StripeAccountEnvelope` first then `StripeNullAccountEnvelope` in case of failure? Just a slight optimization to decode once for most of the cases 🙂

Reply:
See below. This code was needed for older versions of the WooCommerce Payments extension, but is not needed here. Removed in https://github.com/woocommerce/woocommerce-ios/commit/aba8325b9081168679307c777d8fddafef26e03f